src/core/CoolBoard.cpp

```diff
@@ -115,6 +118,8 @@ void CoolBoard::loop() {
   JsonObject &state = root.createNestedObject("state");
   JsonObject &reported = state.createNestedObject("reported");
 
+  reported["Firmware Version"] = (String)COOL_FW_VERSION;
```

Comment:
1. Add that to the `readBoardData()` function
2. **JSON keys must be `camelCase`, so `firmwareVersion` here**